sap-s4hana-soap/2.1/modules/ROOT/pages/sap-s4hana-soap-connector-studio.adoc

```diff
@@ -0,0 +1,175 @@
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for SAP S/4HANA SOAP Connector, follow these steps:
+
+. In the *Mule Palette* view, select *SAP S4 HANA Cloud SOAP* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Global Configuration Elements
+
+To use SAP S/4HANA SOAP Cloud Connector in a Mule app, you can configure a global SAP S/4HANA configuration element for use by SAP S/4HANA Connector operations in your application.
+
+You can put placeholder values in the `mule-artifact.content` configuration file in the `src/main/resources` folder of your project.
+
+You can either enter your credentials in the global configuration content or
+reference a configuration file that contains the values for your credentials.
+Using a configuration file is recommended for simpler maintenance and better reuse of your project. Keeping these values in a separate file is useful if you
+need to deploy to different environments, such as production, development, and QA, where access credentials might differ.
+
+
+=== Configure the SAP S/4HANA Cloud SOAP Config
+
+When you configure a connector, it is best to configure a global element that can be used by all instances of that connector in the app. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target SAP S/4HANA system. SAP S/4HANA Connector supports both Basic authentication and OAuth 2.0 client credentials authentication.
+
+To configure the global element for SAP S/4HANA SOAP Connector, follow these steps:
+
+. In your project, click the *Global Elements* tab at the base of the canvas.
+. In the *Global Configuration Elements* screen, click *Create*:
++
+image::sap-hana-global-config-wizard.png[]
++
+. In the *Choose Global Type* wizard, expand *Connector Configuration*, select *SAP S/4HANA Cloud SOAP Config*, and click *OK*.
+. In the *General* tab, select the connection authentication from the *Connection* drop-down: +
+* <<basic_authentication,*Basic authentication*>> +
+* <<oauth-client-credentials,*OAuth 2 Client Credentials Authentication*>> 
+. Enter the required information for the selected connection type, and click *OK*.
```

Comment:
Kim would remove the comma. We may have to correct this in the template and script, but it is not that important.